comp/logs/agent/config/config.go

```diff
@@ -217,6 +221,7 @@ func BuildHTTPEndpointsWithConfig(coreConfig pkgConfig.Reader, logsConfig *LogsC
 	} else {
 		main.Version = EPIntakeVersion1
 	}
+	main.UseSSL = pointer.Ptr(defaultNoSSL)
```

Comment:
NIT: maybe we can move this into the `main := Endpoint{` block similar to `buildTCPEndpoints`? (also can do `&defaultNoSSL` and omit the pointer util)